alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/FsmAutoConfiguration.java

```diff
@@ -49,58 +55,100 @@
   @Value("${alpha.feature.akka.channel.memory.size:-1}")
   int memoryEventChannelMemorySize;
 
+  @Value("${alpha.feature.akka.transaction.repository.elasticsearch.batchSize:100}")
+  int repositoryElasticsearchBatchSize;
```

Comment:
I think we can use [@ConfigurationProperties][1] to provide these properties, and with the help of [spring-boot-configuration-processor][2], IDE can do auto-completion and show doc on property when we write property files.

[1]: https://docs.spring.io/spring-boot/docs/2.1.6.RELEASE/reference/html/boot-features-external-config.html#boot-features-external-config-vs-value
[2]: https://docs.spring.io/spring-boot/docs/2.1.6.RELEASE/reference/html/configuration-metadata.html#configuration-metadata-annotation-processor